manage_breast_screening/assets/sass/main.scss

```diff
@@ -43,3 +43,7 @@ a[href="#"] {
   right: 0;
   text-align: right;
 }
+
+p:has(+ .app-inset-text) {
+  margin-bottom: 0;
+}
```

Comment:
The default margins are fairly huge aren't they 😣 

But would you mind just adding `nhsuk-u-margin-top-0` to the inset text instead?

Otherwise this problem will come back again should another element (e.g. a list) take the place of the paragraph

i.e. It's not the paragraph's responsibility

Reply:
Thanks @MatMoore

Hadn't realised we'd already overriden the default margins, yeah not a blocker